tests/storage/mpathconf_test.py

```diff
@@ -49,6 +49,21 @@
 }
 """
 
+MPATH_CONF = f"""\
+{mpathconf.CURRENT_TAG}
+# Valid configuration
+"""
+
+MPATH_CONF_PRIVATE = f"""\
+{mpathconf.CURRENT_TAG}
+{mpathconf._PRIVATE_TAG}
+"""
+
+MPATH_CONF_OLD_TAG = f"""\
+{mpathconf._OLD_TAGS[-1]}
+# Invalid configuration
```

Comment:
Are you sure this is invalid? This is just old revision that needs an upgrade.

Reply:
Right, is invalid in the sense that it would return a NO in the validity check in the `vdsm-tool`. I'll use more specific wording here.